3denEnhanced/stringtable.xml

```diff
@@ -3207,6 +3209,7 @@
     </Key>
     <Key ID="STR_ENH_FUNCTIONSVIEWER_RECOMPILEALL_TEXT">
         <English>Recompile All</English>
+		<Italian>Ricompila Tutto</Italian>
```

Comment:
some bad TABs are still here 😈

Reply:
I know, I'll fix them completely once the Italian translation is fully done. :)